packages/paste-website/src/components/SlantedBackgroundGradient.tsx

```diff
@@ -3,51 +3,60 @@ import {BackgroundColorOptions} from '@twilio-paste/style-props';
 import {useTheme} from '@twilio-paste/theme';
 import {useWindowSize} from '../hooks/useWindowSize';
 
-const SKEW_ANGLE = -0.15708; // radians. -9deg
 // https://codepen.io/enbee81/full/yLyrmyg
-const getSkewOffset = (width: number): number => Math.ceil((Math.tan(SKEW_ANGLE) * width) / 2);
+const getSkewOffset = (width: number, skewAngle: number): number => Math.floor((Math.tan(skewAngle) * width) / 2);
+
+// -9deg in radians.
+export const HOMEPAGE_SKEW_ANGLE = -0.15708;
 
 // Make this shareable between several components implementing skew (footer, hero)
-export function useSlantedSkew(): [number] {
+export function useSlantedSkew(angle: number | undefined = HOMEPAGE_SKEW_ANGLE): [number] {
   const {width = 1740} = useWindowSize();
-  return [getSkewOffset(width)];
+  return [getSkewOffset(width, angle)];
 }
 
 interface BackgroundGradientProps {
-  angle?: string;
+  skewAngle?: number;
+  gradientAngle?: string;
   startColor: BackgroundColorOptions;
   endColor: BackgroundColorOptions;
+  styles?: {};
 }
 
 export const SlantedBackgroundGradient: React.FC<BackgroundGradientProps> = ({
-  angle,
   startColor,
   endColor,
+  skewAngle = HOMEPAGE_SKEW_ANGLE,
+  gradientAngle = '90deg',
+  styles,
   ...props
 }) => {
-  const [skewOffset] = useSlantedSkew();
+  const [skewOffset] = useSlantedSkew(skewAngle);
   const {backgroundColors} = useTheme();
 
   return (
     <div
-      {...props}
+      aria-hidden
```

Comment:
I think this is hiding the entire hero section from screen readers. I can't get to it at all on this branch